scripts/linter/plugins/valid-specs.js

```diff
@@ -0,0 +1,53 @@
+const visit = require("unist-util-visit");
+const yaml = require("js-yaml");
+const fs = require('fs');
+const path = require('path');
+
+const ROOT = path.join(__dirname, "..", "..", "..");
+const dataDir = path.resolve(ROOT, 'content', 'data');
+const specMap = yaml.safeLoad(fs.readFileSync(path.join(dataDir, 'specifications.yaml'), 'utf8'));
+
+const ruleId = "stumptown-linter:valid-specifications";
+const specURLRegex = new RegExp("^http(s)?://[^#]+#.+");
+
+/**
+ * Check the front-matter for valid `specifications`
+ */
+function attacher() {
+    return function transformer(tree, file) {
+        visit(tree, "yaml", node => {
+            let specs = node.data.yaml.specifications;
```

Comment:
```suggestion
            const specs = node.data.yaml.specifications;
```

Reply:
I'm actually modifying `specs` to be always an array in line 24, so I think this should stay `let`.

Reply:
Oh I missed that `specs` is reassigned below. Disregard!